tests/integration/tests/test-home-page.js

```diff
@@ -0,0 +1,80 @@
+/* global browser */
+/* global $ */
+/* global $$ */
+
+"use strict";
+
+const assert = require("assert");
+const HomePage = require("../pages/desktop/home.page");
+const NavBar = require("../regions/navbar.region");
+const UserDashboardPage = require("../pages/desktop/dashboard.page");
+
+describe("Firefox Monitor homepage", function() {
+  //this.retries(2);
```

Comment:
Should this be uncommented (like in L7 of previous file)?

Reply:
I am thinking retries aren't so bad, as there are multiple things that could go wrong. What do you think @pdehaan

Reply:
:+1: on retries, and 2 is a good number.